### ISSUE TITLE
No timer sound at end of period on Android 4.4.4 (but works on 2.3.7)

### ISSUE THREAD

**Issue by msiep** [yuttadhammo/BodhiTimer#5]
My old phone has Android 2.3.7 (CyanogenMod-7..2.0-glacier) and BodhiTimer works fine on it, but my new phone has Android 4.4.4 (CyanogenMod 11-20141112-SNAPSHOT-M12-jflte) and there's no sound at the end of the timed period, even though it works fine when I tap "Test Sound" in the settings.  I've tried with several different sound options, e.g. Three Bells, Singing Bell, System Sound.  (The Start Sound does work, however.)

**Comment by msiep** [comment]
This is with version 4.1.3 of Bodhi Timer, installed from F-Droid.org on both devices.

**Comment by msiep** [comment]
I discovered this was not attributable to BodhiTimer but was a symptom of a more general problem my phone had, which also caused the clock to fail to stay updated (e.g. wake phone, see time it was several minutes ago, turn off then on, see correct time) and clock alarms to fail to go off.  I managed to fix it by doing a fresh install of CyanogenMod, but still have no idea what caused it in the first place.
